src/object.d

```diff
@@ -3241,6 +3241,7 @@ template RTInfo(T)
     enum RTInfo = null;
 }
 
+// Compare floating point numbers for ordering (one instantiation per width).
```

Comment:
Ah, OK - it doesn't apply to floats because the compiler strips head const on call. Only types with indirections.